crates/compiler/gen_dev/src/generic64/x86_64.rs

```diff
@@ -942,15 +942,21 @@ impl CallConv<X86_64GeneralReg, X86_64FloatReg, X86_64Assembler> for X86_64Windo
             storage_manager.claim_stack_area(dst, layout_interner.stack_size(*ret_layout));
             todo!("claim first parama reg for the address");
         }
-        for (i, (sym, layout)) in args.iter().zip(arg_layouts.iter()).enumerate() {
+
```

Comment:
Do these changes not apply to load_args above? Do we not use load_args above anymore and instead use load_arg_general and load_arg_float?

We should probably update it as well or remove it.